MonoGame.Framework/Graphics/States/Blend.cs

```diff
@@ -58,11 +58,11 @@ public enum Blend
         /// </summary>
 		InverseDestinationAlpha,
 	    /// <summary>
-        /// Each component of the color is multiplied by a constant in the <see cref="GraphicsDevice.BlendFactor"/>.
+        /// Each component of the color is multiplied by a constant in the GraphicsDevice.BlendFactor.
 	    /// </summary>
 		BlendFactor,
         /// <summary>
-        /// Each component of the color is multiplied by a inversed constant in the <see cref="GraphicsDevice.BlendFactor"/>.
+        /// Each component of the color is multiplied by a inversed constant in the GraphicsDevice.BlendFactor.

```

Comment:
I suspect that doc string is wrong...  it think meant to say `BlendState.BlendFactor`:

https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Graphics/States/BlendState.cs#L148